docs/release/trg-0/trg-8-03.md

```diff
@@ -0,0 +1,257 @@
+---
+title: TRG 8.03 - KIT Artefacts
```

Comment:
Typo "Artifacts" instead of "Artefacts"

Reply:
Are we talking British or American English? :D